### PR TITLE
INGK-533 Interface network class soem service

### PR DESCRIPTION
**Pull request by MartinALL91**

### CODE REVIEW

**Review by WoodElv — APPROVED** [review]

File: ingenialink/eoe/network.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    STOP = '4'
+
+
+class EoENetwork(EthernetNetwork):
+    """Network for EoE (Ethernet over EtherCAT) communication.
+
+        Args:
+        ifname (str): Network interface name.
+        connection_timeout (float): Time in seconds of the connection timeout
+        to the EoE service.
```
Comment (GullinBustin) [inline ingenialink/eoe/network.py:24]: The docstring format is not right, try to use black

**Review by GullinBustin — CHANGES_REQUESTED** [review]
Add unit test

**Comment by GullinBustin** [comment]
I just see the tests in https://github.com/ingeniamc/ingenialink-python/pull/234#pullrequestreview-1237652189

File: ingenialink/eoe/network.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        self._eoe_service_started = False
+
+    def connect_to_slave(self, target, dictionary=None, port=1061,
+                         connection_timeout=DEFAULT_ETH_CONNECTION_TIMEOUT,
+                         servo_status_listener=False,
+                         net_status_listener=False):
+        self._configure_slave(target)
+        if not self._eoe_service_started:
+            self._start_eoe_service()
+        return super().connect_to_slave(target, dictionary,
```
[the remark this answers is not in the document]
Comment (MartinALL91) [reply ingenialink/eoe/network.py]: To enable multi-drive communication.

**Review by GullinBustin — APPROVED** [review]

**Pull request merged**